### DISCUSSION TITLE
Oxalis 5.0.5

### DISCUSSION THREAD

**Discussion by aaron-kumar** [OxalisCommunity/oxalis#536] [General]
* Upgraded vefa-peppol version to 2.0.2 : ec6c95c683b7232c013039c2822b254d3d447487
* Mask secrets by @johnksv : 7a5c44bc0f6880d6c80e5b149a9f050fe401cb77 and 9d5d460d516bc1be27da14a4a2cc625dedcb4009 👏
* Documentation to configure correct trusted certificate list : aa359fed2c9842502b478dae32a463facb9d4cdc  👏
* Guava version upgrade from 28.2-jre to 30.0-jre : 743ef2a284927fb8580b244c1479eec9ec32f011 and d2e137dfe8cf207416e4dcee10ada19583e8368f
* Bump httpclient from 4.5.11 to 4.5.13 : 5fd7ccde5fa2d82e340e78011948968ac3b353ac
* Bump jetty-server from 9.4.35.v20201120 to 9.4.41.v20210516 : ab033fbc2e8ab2c1bb8f450aa7fc7b98efe0cc14
* Updated openjdk:8u191 to openjdk:8u212 due to due to missing manifest file : 4aa628612bcd6b9e4548cd2ccc514db56453c8c6

<hr /><em>This discussion was created from the release <a href='https://github.com/OxalisCommunity/oxalis/releases/tag/v5.0.5'>Oxalis 5.0.5</a>.</em>

**Comment by przemyslawss** [comment]
Is it the latest version available on Maven central when and where?

**Comment by aaron-kumar** [comment]
@przemyslawss yes, here : https://search.maven.org/search?q=g:network.oxalis%20AND%20a:oxalis

**Reply by przemyslawss** [reply]
Thanks :)

**Reply by przemyslawss** [reply]
After updating oxalis I have many issues that are hard to solve, any tips how to resolve this issue?

network.oxalis.api.lang.OxalisLoadingException: Unable to detect mode.

**Reply by aaron-kumar** [reply]
To reproduce issue, please provide following information:
1. Are you using API version i.e. downloading artifacts from maven central and adding them as dependency? OR Are you using standalone version provided as part of distribution e.g https://github.com/OxalisCommunity/oxalis/releases/  or https://github.com/OxalisCommunity/Oxalis-AS4/releases ?
2. Are you running it in Docker container?
3. Upgrade TO which version? i.e. Oxalis 5.0.5  and Oxalis-AS4 5.0.3? 
4. Upgrading FROM version? i.e. Oxalis and Oxalis-AS4 version

**Reply by przemyslawss** [reply]
There is 5.0.3, but we managed to fix it by changing dependencies on bouncycastle import that was used by as4
